cli/reporter.py

```diff
@@ -226,9 +230,87 @@ def clean_read(ifile):
                         break  # EOF
                     errors.append({'property': line, 'description': line2})
         self.results['metadata_check'] = {
-            'report_path': metadata_check_file,
+            'json_report_path': metadata_check_file,
             'json_errors': errors
         }
+        self.convert_metadata_validation_results()
+        self.write_converted_metadata_reslts()
+
+
+
+    def _parse_metadata_property(self, property_str):
+        if property_str.startswith('.'):
+            return property_str.strip('.'), None, None
+        match = re.match(r'/(\w+)(/(\d+))?(\.(\w+))?', property_str)
+        if match:
+            return match.group(1), match.group(3), match.group(5)
+        else:
+            logger.error(f'Cannot parse {property_str} in JSON metadata error')
+            return None, None, None
+
+    def convert_metadata_validation_results(self):
+        config_file = os.path.join(ETC_DIR, "spreadsheet2json_conf.yaml")
+        with open(config_file) as open_file:
+            xls2json_conf = yaml.safe_load(open_file)
+
+        self.results['metadata_check']['spreadsheet_errors'] = []
+        for error in self.results['metadata_check']['json_errors']:
+            sheet_json, row_json, attribute_json = self._parse_metadata_property(error['property'])
+            sheet = self._convert_metadata_sheet(sheet_json, xls2json_conf)
+            row = self._convert_metadata_row(sheet, row_json, xls2json_conf)
+            column = self._convert_metadata_attribute(sheet, attribute_json, xls2json_conf)
+            if row_json is None and attribute_json is None:
+                new_description = f'Sheet "{sheet}" is missing'
+            elif row_json is None:
+                new_description = f'In sheet "{sheet}", column "{column}" is not populated'
+            elif attribute_json and column:
+                new_description = f'In sheet "{sheet}", row "{row}", column "{column}" is not populated'
+            else:
+                new_description = error["description"].replace(sheet_json, sheet)
+            if column is None:
+                # We do not know this attribute. It's most likely about bioSampleObject
+                continue
+            if 'schema' in new_description:
+                # This is an error specific to json schema
+                continue
+            self.results['metadata_check']['spreadsheet_errors'].append({
+                'sheet': sheet, 'row': row, 'column': column,
+                'description': new_description
+            })
+
+    def write_converted_metadata_reslts(self):
```

Comment:
Judgment call: 
```suggestion
    def write_user_friendly_metadata_results(self):
```

Reply:
I tried to make the function name clearer